examples/form.php

```diff
@@ -40,14 +40,22 @@
 	] // according to Valitron doc.
 ];
 
-$mailerConfig = [
-	'mailer'   => MailHandler::USE_PHPMAILER, // (or USE_POSTMARKAPP, USE_MANDRILL)
+// PhpMailer config.
+/*$mailerConfig = [
+	'mailer'   => MailHandler::USE_MANDRILL, // (or USE_POSTMARKAPP, USE_MANDRILL)
 	'host'     => 'smtp.gmail.com',
 	'user'     => 'YOUR EMAIL',
 	'password' => 'YOUR PASSWORD',
 	'protocol' => 'tls',
 	'port'     => 587,
 	'attachmentsSizeLimit' => 8000000, // around 8MB.
+];*/
+
+// Mandrill config.
+$mailerConfig = [
+	'mailer'   => MailHandler::USE_MANDRILL, // (or USE_POSTMARKAPP, USE_MANDRILL)
+	'password' => '_5mPSvb39BQqnA7G_dOaAA',
+	'attachmentsSizeLimit' => 8000000, // around 8MB.
 ];
```

Comment:
I prefer to have another example for mandrill to keep files simple.

Let's rename files as:

* form2email-basic.php
* form2email-mandrill.php